[2024-07-30T10:26:52.051+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T10:26:52.103+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipelinev5.load_dimension_tables.load_users_dim_table scheduled__2018-11-07T00:00:00+00:00 [queued]>
[2024-07-30T10:26:52.137+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipelinev5.load_dimension_tables.load_users_dim_table scheduled__2018-11-07T00:00:00+00:00 [queued]>
[2024-07-30T10:26:52.138+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T10:26:52.280+0000] {taskinstance.py:2330} INFO - Executing <Task(LoadDimensionOperator): load_dimension_tables.load_users_dim_table> on 2018-11-07 00:00:00+00:00
[2024-07-30T10:26:52.293+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1003) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-30T10:26:52.295+0000] {standard_task_runner.py:63} INFO - Started process 1010 to run task
[2024-07-30T10:26:52.300+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipelinev5', 'load_dimension_tables.load_users_dim_table', 'scheduled__2018-11-07T00:00:00+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpna6lomyy']
[2024-07-30T10:26:52.307+0000] {standard_task_runner.py:91} INFO - Job 118: Subtask load_dimension_tables.load_users_dim_table
[2024-07-30T10:26:52.518+0000] {task_command.py:426} INFO - Running <TaskInstance: pipelinev5.load_dimension_tables.load_users_dim_table scheduled__2018-11-07T00:00:00+00:00 [running]> on host c3b60429ae20
[2024-07-30T10:26:53.023+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipelinev5' AIRFLOW_CTX_TASK_ID='load_dimension_tables.load_users_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2018-11-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2018-11-07T00:00:00+00:00'
[2024-07-30T10:26:53.038+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T10:26:53.126+0000] {load_dim_operator.py:26} INFO - Loading dimension table users
[2024-07-30T10:26:53.228+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:53.347+0000] {sql.py:487} INFO - Running statement: 
                BEGIN;
                TRUNCATE TABLE users;
                
        INSERT INTO users (user_id, first_name, last_name, gender, level)
        SELECT DISTINCT ON (userid) userid, firstname, lastname, gender, level
        FROM staging_events
        WHERE page='NextSong'
        ON CONFLICT (user_id) DO UPDATE
        SET first_name = EXCLUDED.first_name,
            last_name = EXCLUDED.last_name,
            gender = EXCLUDED.gender,
            level = EXCLUDED.level
    ;
                COMMIT;
            , parameters: None
[2024-07-30T10:26:54.184+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T10:26:54.216+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipelinev5, task_id=load_dimension_tables.load_users_dim_table, run_id=scheduled__2018-11-07T00:00:00+00:00, execution_date=20181107T000000, start_date=20240730T102652, end_date=20240730T102654
[2024-07-30T10:26:54.316+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T10:26:54.353+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-30T10:26:54.384+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
